### ISSUE TITLE
Needs better on-screen instructions

### ISSUE THREAD

**Issue by rmscheller** [LANDIS-II-Foundation/LANDVIZ#24]
Currently users must have the Users Guide at hand.  We need to embed instructions, either as a pop-up help menu or simple instructions on the splash screen or quick access to the User Guide PDF.  Or some combination.

**Comment by mslucash** [comment]
I think the tutorial and balloons on this tool by Dominque et al are really good.  

https://climate.databasin.org/

![databasin_clip](https://user-images.githubusercontent.com/7819936/30930882-e498e7d2-a377-11e7-968a-7cb56f1b04ba.JPG)

**Comment by mshukuno** [comment]
Is this related to #23?

**Comment by rmscheller** [comment]
No.  The web site would contain more general information about the project, partners, funding, etc.  Maybe some more detail about how scenarios were constructed, some photos.  The problem is that a LANDViz web site by itself lacks context.
